pagopa/io-dev-api-server
# feat: [IOBP-964] Add zendesk payment subcategories

> [!WARNING]
> this PR depends on https://github.com/pagopa/io-services-metadata/pull/892

## Short description
This pull request add new subcategories for payment issues in both JSON files and updating the router to handle these new subcategories.

## List of changes proposed in this pull request
- Added new subcategories for various payment-related issues as defined in `io-service-metadata`
- Added detailed descriptions for the new payment-related subcategories

## How to test
- After login tap on the top right question marker
- Tap on `Apri una richiesta`
- Tap on `Pagamenti PagoPa`
- Verify that subcategories are displayed
- Tap on each of them and verify if is possible to send a message

https://github.com/user-attachments/assets/960b858e-ece1-408c-81e7-fba9a5c4b2fa
